src/content/en/2020/pwa.md

```diff
@@ -22,3 +22,257 @@ queries: 14_PWA
 #featured_stat_label_3: TODO
 unedited: true
 ---
+
+PWA
+
+Web Almanac 2020
+
+![](https://lh6.googleusercontent.com/SGZ4UHI5jM7J5kLzetmvz9_R9bNQVvpC5qhZYwgpHPu3zuyxo3topCbo3ZINjO-l7_n4aWMEiPEy9XjrfpO3UVX25qcJ22SHI0L92jlaYtSQPfnBVLMjHsYtJ8Fa-goTLccXVgFk)
+
+
+# Introduction
+
+In 1990 we had the first ever browser called the “WorldWideWeb” and ever since the web and the browser has been evolving and for web to progress itself into a native behaviour is a big win special in this ear of mobile domination, URLs have provided an ubiquitous way to distribute information and nothing better than the browser to handle them and hence a technology which provides such capabilities to the browser becomes a game changer and Progressive Web Apps provide such advantages for the web to win yet again.
+
+Simply put, a web application which progresses into a native-like application can be considered as a PWA, built using common web technologies including HTML, CSS and JavaScript and can operate seamlessly across devices and environments on a standards-compliant browser.
+
+The crux of a progressive web app is “The ServiceWorker”, which can be thought of as a proxy sitting between the browser and user and gives the developer total control over the network, rather than the network controlling the application.
+
+It all started in December 2014 when Chrome 40 first implemented the meat of what is now known as Progressive Web Apps which was a collaborative effort for the web standards body and the terms was coined by [Frances Berriman and Alex Russell](https://infrequently.org/2015/06/progressive-apps-escaping-tabs-without-losing-our-soul/) in 2015.
+
+In this chapter of Web Almanac we will be looking into each of the ingredients that make PWA what it is from a data-driven perspective.
+
+  
+
+# Service workers
+
+## Service worker registrations and installability
+
+From the data we gathered it was derived that about **0.88%** for desktop, **0.87%** for mobile for the month of August 2020 it's 49,305 (out of 5,593,642) for desktop and 55,019 (out of 6,347,919) for mobile. It is important that we realize that it translates into **16.6%** of the entire [traffic]
+(https://www.chromestatus.com/metrics/feature/timeline/popularity/990). 
+
+![](https://lh3.googleusercontent.com/LWvKIzfDAY5487x_scblRbPz2mog_xHmRXGth0J49jXrivPqmVZiaFeMgixG8pKxC0bafdb4MBR8yuOk8cc0jxVanEWNGV-QIi-D0xLOGwixg9sWQqQC-Fd12Pg-SIhCxiSrlNSH "Chart")
+
+
+# Insights from lighthouse scores
+
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) provides automated auditing, performance metrics, and best practices for the web and has been instrumental in shaping web’s performance.  The score we gathered over 6811475 pages, has given us great insights on a few important touch points for the month of September 2020.
+
+| Lighthouse Audit         | Weight | Number of Pages | Percentage |   |
```

Comment:
Here's an example from the Figures Guide how to include a table as a figure: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide#table-examples